ci/checksizes.sh

```diff
@@ -45,11 +45,11 @@ case $FULLVER in
     REVERSI_PREOPT_GZ_EXPECTEDSIZE=71000
     REVERSI_OPT_GZ_EXPECTEDSIZE=32000
     ;;
-  2.12.0-M4)
-    REVERSI_PREOPT_EXPECTEDSIZE=518000
-    REVERSI_OPT_EXPECTEDSIZE=121000
-    REVERSI_PREOPT_GZ_EXPECTEDSIZE=70000
-    REVERSI_OPT_GZ_EXPECTEDSIZE=32000
+  2.12.0-M5)
+    REVERSI_PREOPT_EXPECTEDSIZE=526000
+    REVERSI_OPT_EXPECTEDSIZE=124000
+    REVERSI_PREOPT_GZ_EXPECTEDSIZE=67000
+    REVERSI_OPT_GZ_EXPECTEDSIZE=31000
```

Comment:
That's surprising. Significantly more code in non-gzipped versions but significantly less in gzipped versions :-s I'm especially worried that fastopt.js is 8 KB larger. Could you post a diff of the two in a gist, please?